### ISSUE TITLE
Add UNITE paper from CVPR 2024

### ISSUE THREAD

**Issue by reddyav1** [xuyu0010/awesome-video-domain-adaptation#5]
Hello!

I would like to kindly request the addition of the following paper to the closed-set VDA section:

Title: Unsupervised Video Domain Adaptation with Masked Pre-Training and Collaborative Self-Training (UNITE)

Arxiv: https://arxiv.org/abs/2312.02914

CVPR: https://openaccess.thecvf.com/content/CVPR2024/papers/Reddy_Unsupervised_Video_Domain_Adaptation_with_Masked_Pre-Training_and_Collaborative_Self-Training_CVPR_2024_paper.pdf

Code: https://github.com/reddyav1/unite

Thanks for maintaining this list!

**Comment by xuyu0010** [comment]
Hi Arun,

Sorry for the delayed reply, the paper has been added.

Regards
Yuecong
